spec/rails_helper.rb

```diff
@@ -4,8 +4,7 @@
 require File.expand_path("../../config/environment", __FILE__)
 require 'rspec/rails'
 require 'capybara/rails'
-
-
+require_relative 'support/database_cleaner'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.